src/client/pages/Geo/GeoMap/GeoMapMenuRecipes/index.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
This doesn’t look good , maybe better to rename the component itself in upper case ?

Reply:
Yes, I don't know why the precommit hook was complaining about this. I haven't created this, I didn't want to modify it. But will do it.

Reply:
Yes I know you didn't write this component, but let's fix it anyway